phpBB/phpbb/install/converter/controller/index.php

```diff
@@ -0,0 +1,164 @@
+	 * @param \phpbb\template\template                                                                                   $template
+	 * @param \phpbb\install\module_interface                                                                            $module
+	 * @param \phpbb\install\helper\config|\phpbb\install\installer_configuration                                        $install_config
+	 * @param \phpbb\install\helper\iohandler\factory|\phpbb\install\helper\iohandler\iohandler_interface                $iohandler
+	 * @param \phpbb\request\request_interface                                                                           $request
+	 * @param                                                                                                            $phpbb_root_path
+	 * @param                                                                                                            $php_ext
+	 */
+	public function __construct(\phpbb\install\helper\container_factory $container_facotry,\phpbb\install\converter\controller\helper $helper, \phpbb\install\helper\navigation\navigation_provider $nav_provider, \phpbb\language\language $language, \phpbb\template\template $template, \phpbb\install\module_interface $module, \phpbb\install\helper\config $install_config, \phpbb\install\helper\iohandler\factory $iohandler, \phpbb\request\request_interface $request, $phpbb_root_path, $php_ext)
+	{
+		$this->container_factory = $container_facotry;
+		$this->helper = $helper;
+		$this->menu_provider = $nav_provider;
+		$this->language = $language;
+		$this->template = $template;
+		$this->config_php_file = new config_php_file($phpbb_root_path,$php_ext);
+		$this->module = $module;
+		$this->install_config = $install_config;
+		$this->iohandler_factory = $iohandler;
+		$this->request = $request;
+		$this->helper->handle_language_select();
+	}
+
+	/**
+	 * @return \Symfony\Component\HttpFoundation\Response|\Symfony\Component\HttpFoundation\StreamedResponse
+	 */
+	public function handle()
+	{
+		if ($this->request->is_ajax())
+		{
+			$this->iohandler_factory->set_environment('ajax');
+			$ajax_handler = $this->iohandler_factory->get();
+			$module = $this->module;
+			$module->setup($this->install_config,$ajax_handler);
+			$response = new StreamedResponse();
+			$response->setCallback(function() use ($ajax_handler, $module){
+				/*The lock must be the first thing to be acquired as the js queries every 250ms for status
+				and if we acquire the lock later the js may issue another request before previous completes
+				thus stuck in an infinite loop of continue -> lock not acquired -> again continue ....
+				*/
+				$ajax_handler->acquire_lock();
+//				$ajax_handler->add_success_message('Database Configuration Completed',array(
+//					'CONVERTER_START',
+//					'#',
+//				));
+//				$ajax_handler->send_response(true);
+				$module->run();
```

Comment:
Thats what I have been saying that I dont use the installer.php I run the module directly. Wont installer.php run would cause alll the installer modules to run?